site/src/pages/WorkspaceSettingsPage/WorkspaceParametersPage/WorkspaceParametersPage.test.tsx

```diff
@@ -73,3 +73,58 @@ test("Submit the workspace settings page successfully", async () => {
 		});
 	});
 });
+
+test("Submit button is only enabled when changes are made", async () => {
```

Comment:
I think for now, having a test for this in Jest is ok, but in the future, you can try to use [storybook and interaction tests](https://storybook.js.org/tutorials/ui-testing-handbook/react/en/interaction-testing/ ) :)